accern_xyme/accern_xyme.py

```diff
@@ -1470,19 +1468,21 @@ def __init__(self, pipeline: PipelineHandle, data_id: str) -> None:
         self._data_id = data_id
         self._value: Optional[ByteResponse] = None
 
-    def maybe_value(self) -> Optional[ByteResponse]:
+    def has_fetched(self) -> bool:
+        return self._value is not None
+
+    def get(self) -> Optional[ByteResponse]:
         if self._value is None:
```

Comment:
if not self.has_fetched()

Reply:
sure

Reply:
no we need that for the type checker to know it is not None

Reply:
okay